src/pocketmine/level/Level.php

```diff
@@ -1223,8 +1223,8 @@ public function getFullLight(Vector3 $pos) : int{
 		$chunk = $this->getChunk($pos->x >> 4, $pos->z >> 4, false);
 		$level = 0;
 		if($chunk instanceof Chunk){
-			//Do not decrease skylightlvl over day. MCPC and MCPE don't do this either.
-			$level = max($chunk->getBlockLight($pos->x & 0x0f, $pos->y & Level::Y_MASK, $pos->z & 0x0f), $chunk->getBlockSkyLight($pos->x & 0x0f, $pos->y & Level::Y_MASK, $pos->z & 0x0f));
+			//Do not decrease skylightlvl over time. MCPC and MCPE don't do this either.
```

Comment:
It exists because i simply modified https://github.com/pmmp/PocketMine-MP/blob/master/src/pocketmine/level/Level.php#L1234. But now that you say it, I probably should just get rid of it.